sql/lexer_test.go

```diff
@@ -123,3 +123,16 @@ func TestLexSQL(t *testing.T) {
 		a.Equal(vals[i], n.val)
 	}
 }
+
+func TestLexFuncSQL(t *testing.T) {
+	a := assert.New(t)
+	slct := "SELECT func(func2(\"arg0\", arg1), arg_2) AS (info, score) FROM a_table where a_table.col_1 > 100;"
```

Comment:
Maybe `AS` can also be used as standard sql aliases: https://www.w3schools.com/sql/sql_ref_as.asp, can add another case to test this case so that the parser will be a little more general.

Reply:
I left a TODO comment, will implement it in the next PR, and maybe have some discussion with https://github.com/sql-machine-learning/sqlflow/pull/591